### ISSUE TITLE
Plug-in system

### ISSUE THREAD

**Issue by agershun** [agershun/alasql#166]
I am thinking about splitting AlaSQL on core and plug-ins (from- and into- functions, database backends, graph functionality, some libraries). 

Now AlaSQL has ```REQUIRE``` command wich supports Node, browser and webworkers, so it could be:
```
    alasql('REQURE XSLX, GRAPH,"/lib/mylib.js"');
    alasql('SEARCH VERTEX INTO XLSX("mygraph.xlsx") FROM GEDF("mygraph.gedf")');
```
Question 1: There are many plug-ins systems (e.g. jQuery, require, etc.). What plug-in system would be better for AlaSQL?
Question 2: What minimal functionality is necessary in the core?

**Comment by agershun** [comment]
Done. Will be available in version 0.1.6 or in the develop branch

**Comment by mathiasrw** [comment]
:+1:
